cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -87,7 +90,10 @@ func setWeakHostOnInterface(ipAddress string) error {
 
 	log.Printf("[Azure CNS] Going to enable weak host send/receive on interface: %v", args)
 	c := exec.Command("cmd", args...)
```

Comment:
Shouldn't we lock before this line?

Reply:
nope, that just constructs the command, it's actually run in c.OutPut()